addons/escoria-core/game/core-scripts/migrations/esc_migration_manager.gd

```diff
@@ -0,0 +1,172 @@
+# Class that handles migrations between different game or escoria versions
+extends Node
```

Comment:
Same question here: does it need to be added to a scene, or could it be instanced as an Object instead?

Reply:
Done